app/components/stepper-signup.hbs

```diff
@@ -1,25 +1,62 @@
 <OnboardingCard>
   <form>
+    {{#if (not (eq this.currentStep 0))}}
+      <img src='assets/icons/onboarding-card-rds-logo.png' alt='RDS-Logo' class='signup-details__rds-logo' data-test-rds-logo />
+    {{/if}}
     {{#if (eq this.currentStep 0)}}
-      <SignupSteps::StepZero
-        @currentStep={{this.currentStep}}
-        @letsGoHandler={{this.letsGoHandler}}
+      <SignupSteps::StepZero 
+        @currentStep={{this.currentStep}} 
+        @startHandler={{this.startHandler}} 
       />
-    {{else}}
-      <img
-        src='assets/icons/onboarding-card-rds-logo.png'
-        alt='RDS-Logo'
-        class='signup-details__rds-logo'
-        data-test-rds-logo
+    {{else if (eq this.currentStep 1)}}
+      <SignupSteps::StepOne 
+        @currentStep={{this.currentStep}} 
+        @incrementStep={{this.incrementStep}} 
+      />
+    {{else if (eq this.currentStep 2)}}
+      <SignupSteps::stepTwo 
+        @startHandler={{this.startHandler}} 
+      />
+    {{else if (eq this.currentStep 3)}}
+      <JoinSteps::StepOne 
+        @startHandler={{this.startHandler}} 
+        @setIsPreValid={{this.setIsPreValid}}
+        @isValid={{this.isValid}} 
+        @setIsValid={{this.setIsValid}} 
+      />
+    {{else if (eq this.currentStep 4)}}
+      <JoinSteps::StepTwo 
+        @startHandler={{this.startHandler}} 
+        @setIsPreValid={{this.setIsPreValid}}
+        @isValid={{this.isValid}} 
+        @setIsValid={{this.setIsValid}} 
+      />
+    {{else if (eq this.currentStep 5)}}
+      <JoinSteps::StepThree 
+        @startHandler={{this.startHandler}} 
+        @setIsPreValid={{this.setIsPreValid}}
+        @isValid={{this.isValid}} 
+        @setIsValid={{this.setIsValid}} 
+      />
+    {{else if (eq this.currentStep 6)}}
+      <JoinSteps::StepFour 
+        @startHandler={{this.startHandler}} 
+        @setIsPreValid={{this.setIsPreValid}}
+        @isValid={{this.isValid}} 
+        @setIsValid={{this.setIsValid}} 
       />
-      {{#if (eq this.currentStep 1)}}
-        <SignupSteps::StepOne
-          @currentStep={{this.currentStep}}
-          @incrementStep={{this.incrementStep}}
-        />
-      {{else if (eq this.currentStep 2)}}
-        <SignupSteps::stepTwo @letsGoHandler={{this.letsGoHandler}} />
-      {{/if}}
     {{/if}}
   </form>
-</OnboardingCard>
\ No newline at end of file
+
+  {{#if (not (or (eq this.currentStep 0) (eq this.currentStep 1) (eq this.currentStep 2)))}}

```

Comment:
Can it be simplified like this? I think we're only checking here if step is 2, right?
```suggestion
  {{#if (eq this.currentStep 2)))}}

```

Reply:
here we are checking that the current step should not 0,1and2 so we can't simplified like this